### PR TITLE
[DOC-306] [DOC-320] [DOC-357] [yba] 2024.1 Release management enhancement, 2024.1 version naming changes.

### PR DESCRIPTION
**Pull request by ddhodge**
Release management updates
DOC-306: Update YBA docs for Releases feature
DOC-320: Add 2024.1 version naming changes to docs
DOC-357: New Stable Release numbering (2024.1)
@netlify /preview/yugabyte-platform/manage-deployments/upgrade-software/

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/manage-deployments/upgrade-software.md [hunk 1]
```diff
@@ -28,9 +28,9 @@ When performing a database upgrade, do the following:
 
 1. [Review major changes in previous YugabyteDB releases](../upgrade-software-prepare/). Depending on the upgrade you are planning, you may need to make changes to your automation or upgrade your Linux operating system.
```
Comment (hari90) [inline docs/content/preview/yugabyte-platform/manage-deployments/upgrade-software.md:29]:
```suggestion
1. [Prepare to upgrade a universe](../upgrade-software-prepare/). Depending on the upgrade you are planning, you may need to make changes to your automation or upgrade your Linux operating system.
```

File: docs/content/preview/yugabyte-platform/manage-deployments/upgrade-software-install.md [hunk 2]
```diff
@@ -110,42 +76,62 @@ You can view your Universe ID from your YBA universe URL, as follows:
...
+## Monitor the universe
 
 Once all the nodes have been upgraded, monitor the universe to ensure it is healthy:
 
 - Make sure workloads are running as expected and there are no errors in the logs.
 - Check that all nodes are up and reachable.
 - Check the [performance metrics](../../alerts-monitoring/anywhere-metrics/) for spikes or anomalies.
 
-If you have problems, you can roll back any time.
+If you have problems, you can [roll back](#roll-back-an-upgrade) any time.
```
Comment (hari90) [inline docs/content/preview/yugabyte-platform/manage-deployments/upgrade-software-install.md:87]:
```suggestion
If you have problems, you can [roll back](#roll-back-an-upgrade) during this time.
```

File: docs/content/preview/yugabyte-platform/manage-deployments/upgrade-software-install.md [hunk 3]
```diff
@@ -110,42 +76,62 @@ You can view your Universe ID from your YBA universe URL, as follows:
...
 If you are satisfied with the upgrade:
 
 - For upgrades that do not require finalizing, the upgrade is effectively complete.
 
 - For upgrades that require finalizing, proceed to [Finalize](#finalize-an-upgrade) the upgrade.
 
-### Roll back an upgrade
+## Roll back an upgrade
+
+If you aren't satisfied with an upgrade, you can roll back to the version that was previously installed. The process is similar to [performing an upgrade](#perform-the-upgrade).
```
Comment (hari90) [inline docs/content/preview/yugabyte-platform/manage-deployments/upgrade-software-install.md:99]: The process is similar to

is this necessary?

File: docs/content/preview/yugabyte-platform/manage-deployments/ybdb-releases.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+headcontent: View and import YugabyteDB releases
+menu:
+  preview_yugabyte-platform:
+    identifier: ybdb-releases
+    parent: upgrade-software
+    weight: 15
+type: docs
+---
+
+Before you can upgrade your universe to a specific version of YugabyteDB, verify that the release is available and, if necessary, import the release into YugabyteDB Anywhere.
```
Comment (hari90) [inline docs/content/preview/yugabyte-platform/manage-deployments/ybdb-releases.md:15]:
```suggestion
Before you can upgrade your universe to a specific version of YugabyteDB, verify that the release is available in YugabyteDB Anywhere and, if necessary, import it .
```

File: docs/content/preview/yugabyte-platform/manage-deployments/ybdb-releases.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+You can filter the list by text and release type, and choose to display only allowed releases.
+
+To view the release details, select a release in the list.
+
+To delete or disable a release, click its corresponding **Actions**. Disabled releases are not available when creating universes.
+
+## Import a release
+
+If a release that you want to install on a universe is not available, import it as follows:
```
Comment (hari90) [inline docs/content/preview/yugabyte-platform/manage-deployments/ybdb-releases.md:31]:
```suggestion
If a release that you want to install or upgrade is not available, import it as follows:
```

File: docs/content/preview/yugabyte-platform/manage-deployments/ybdb-releases.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+1. On the **Releases** page, click **New Release** to open the **Import Database Release** dialog as shown in the following illustration:
+
+    ![Import Release](/images/yp/import-releases-2024.png)
```
Comment (hari90) [inline docs/content/preview/yugabyte-platform/manage-deployments/ybdb-releases.md:35]: The image has a lot of white space. Can we trim it?

**Review by hari90 — CHANGES_REQUESTED** [review]

**Review by hari90 — APPROVED** [review]

**Comment by rajmaddy89** [comment]
We need to include the Best Practices from the doc: https://docs.google.com/document/d/1UvYk_dtIZKALteQSvpBuryrn2NyFYyD81yMzJxXdtVM/edit#heading=h.i1si75mqkqn6

File: docs/content/preview/releases/yba-releases/_index.md [hunk 7]
```diff
@@ -55,16 +55,18 @@ For information on YugabyteDB release support timelines, refer to [YugabyteDB re
 
 Keep YBA up-to-date with the latest stable version to get the latest fixes and improvements, as well as to be able to deploy the latest releases of YugabyteDB.
 
-Note that you cannot deploy or upgrade a universe to a version of YugabyteDB that is later than the version of YugabyteDB Anywhere. To upgrade a universe to a more recent version of YugabyteDB, you may first have to upgrade YugabyteDB Anywhere.
+Note that you cannot deploy or upgrade a universe to a higher version of YugabyteDB than the version of YugabyteDB Anywhere. To upgrade a universe to a more recent version of YugabyteDB, you may first have to upgrade YugabyteDB Anywhere.
```
Comment (ddhodge) [inline docs/content/preview/releases/yba-releases/_index.md:58]: You can't use YBA to deploy versions of YugabyteDB that are newer than your YBA instance. To upgrade a universe to a more recent version of YugabyteDB, you may first have to upgrade YBA.

**Comment by aishwarya24** [comment]
Created a new PR - https://github.com/yugabyte/yugabyte-db/pull/22596

**Pull request closed without merging**